File: 3.swift
import Foundation

// Функция для переворота числа и удаления ведущих нулей
func revers_num(_ num: Int) -> Int {
    var num = num
    var revers = 0
    while num != 0 {  // Переворачиваем число
        revers = revers * 10 + num % 10
        num /= 10
    }
    return revers
}

func main() -> Int {
    setlocale(LC_ALL, "")
    
    guard let n = Int(readLine() ?? "") else {
        print("Количество элементов должно быть положительным и целым")
        return 1
    }
    
    // Проверка на корректность ввода
    if n <= 0 {
        print("Количество элементов должно быть положительным и целым")
        return 1
    } 
    else {
        for _ in 0..<n {
            guard let num = Int(readLine() ?? "") else {
                print("Числа должны быть положительным и целым")
                return 1
            }
            
            // Проверка на корректность ввода
            if num <= 0 {
                print("Числа должны быть положительным и целым")
                return 1
            }
            else {
                // Вызываем функцию revers_num и выводим
                print(revers_num(num), terminator: " ")
            }
        }
    }
    
    return 0
}

// Вызываем основную функцию
let _ = main()

File: 3.ts
// Функция для переворота числа
function revers_num(num: number): number {
    let revers = 0;
    while (num !== 0) {
        revers = revers * 10 + num % 10;
        num = Math.floor(num / 10);
    }
    return revers;
}

async function main(): Promise<number> {
    const nInput = prompt("") || "";
    const n = parseInt(nInput);

    // Проверка ввода
    if (isNaN(n) || n <= 0) {
        console.log("Количество элементов должно быть положительным и целым");
        return 1;
    }
    // Чтение чисел
    const numbersInput = prompt("") || "";
    const numbers = numbersInput.trim().split(/\s+/).map(Number);

    let output = "";
    for (let i = 0; i < n; ++i) {
        const num = numbers[i];
        
        if (isNaN(num) || num <= 0) {
            console.log("Числа должны быть положительным и целым");
            return 1;
        }
        
        output += revers_num(num);
        if (i < n - 1) output += " ";// Вывод числа
    }

    console.log(output);
    return 0;
}

// Запуск программы
main();
